.github/workflows/release-check.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    tags:
+      - v0.*.*
+      - v1.*.*
+
+jobs:
+  tests:
+    name: Release sanity tests
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checking code health
+        run: make tools lint test
+
+      - name: Checking the version
+        run: make check-version
```

Comment:
This workflow will ensure that we get notified when a tag was pushed and the version was not updated.